### DISCUSSION TITLE
Support custom output formatters

### DISCUSSION THREAD

**Discussion by modocache** [Quick/Quick#1073] [Ideas]
Luckily, some nerd wrote down [how RSpec implements custom output formatters](http://modocache.svbtle.com/code-reading-rspec-output-formatting).

An example in Objective-C can be found here: https://github.com/kiwi-bdd/Kiwi/pull/449

I wonder if it'd be possible to implement an output formatter that aggregates results and displays an OS X notification with a breakdown of passed, pending, failed, and errored?

**Comment by modocache** [comment]
Parameters passed to formatters should include the number of pending examples, as described in #38. At that point pending examples should stop printing directly via `NSLog`.

**Comment by istx25** [comment]
What's the status on this?

**Comment by modocache** [comment]
This is a sort of "wishlist" feature, I'm not working on it at the moment. :)
